recipes/httpstan/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
```

Comment:
Delete this file as it's not used

Reply:
I'm not understanding why this needs to be removed. It is a differing license than the MIT one from the exmple/meta.yaml and documentation [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file) it seems to suggest I include it. This was the [license](https://github.com/stan-dev/httpstan/blob/main/LICENSE) file found in the source git

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file

blurb comments in the example/meta.yaml
```
  license_family: MIT
  # It is strongly encouraged to include a license file in the package,
  # (even if the license doesn't require it) using the license_file entry.
  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
```

Reply:
conda-build will use the file from the source tarball. you don't need it in the recipe itself.